cmd/account/rotate-secret.go

```diff
@@ -205,14 +205,17 @@ func (o *rotateSecretOptions) run() error {
 
 	// Update osdManagedAdmin secrets
 	// Username is osdManagedAdmin-aaabbb
-	osdManagedAdminUsername := common.OSDManagedAdminIAM + "-" + accountIDSuffixLabel
+	osdManagedAdminUsername := common.OSDManagedAdminIAM
 
 	// Create new access key
-	createAccessKeyOutput, err := awsClient.CreateAccessKey(&iam.CreateAccessKeyInput{
-		UserName: aws.String(osdManagedAdminUsername),
-	})
+	createAccessKeyOutput, err := awsClient.CreateAccessKey(&iam.CreateAccessKeyInput{UserName: aws.String(osdManagedAdminUsername)})
 	if err != nil {
```

Comment:
instead of checking if not nil here, can we check if it is the user not found error?

Reply:
Added a check below.